docs/contributing/gsoc-ideas.md

```diff
@@ -125,43 +123,57 @@ contributions throughout the application process, including after the
 application deadline.
 
 We are more interested in candidates if we see them submitting good
-contributions to Zulip projects, helping other applicants on GitHub
-and in
-[the Zulip development community][development-community],
-learning from our suggestions,
-[trying to solve their own obstacles and then asking well-formed
-questions](https://blogs.akamai.com/2013/10/you-must-try-and-then-you-must-ask.html),
-and developing and sharing project ideas and project proposals that
+contributions to Zulip projects, helping other applicants on [GitHub](https://github.com/zulip/zulip)
+and on [chat.zulip.org](https://chat.zulip.org) (CZO),
```

Comment:
We prefer to always link to https://zulip.com/developer-community/ and avoid abbreviations for it, because the context on that page about how to behave is helpful (and a common problem is the flood of new GSoC students not having read the guidelines).  Also abbreviations are best to avoid since they require extra context to understand, so I try to never type them :)